previews/primer/beta/button_preview/leading_visual.html.erb

```diff
@@ -2,9 +2,22 @@
     scheme: scheme,
     size: size,
     block: block,
-    id: id,
+    id: "button-with-leading-icon-preview",
     align_content: align_content
 )) do |component| %>
   <% component.with_leading_visual_icon(icon: :search) %>
-  Button
+  Leading icon button
+<% end %>
+
+<%= render(Primer::Beta::Button.new(
```

Comment:
Would you mind actually creating a _new_ preview for the SVG version so we can isolate the behavior of each? 🙌

Reply:
For sure! Done in [e6dd38f](https://github.com/primer/view_components/pull/2215/commits/e6dd38f6c17021ae0a405c71e275eb09ab3c491e)